exampleSite/content/en/_includes/_index.md

```diff
@@ -1,3 +1,3 @@
 ---
-GeekdocHidden: true
+geekdocHidden: true
```

Comment:
The casing is intended. Parameters in Hugo are case-insensitive by default, so casing technically doesn't matter, but we decided to use lowercase in the config files and uppercase for occurrences in templates and front matters.

Reply:
Hmm, now I'm not sure either :smile:  Let me think about it a bit

Reply:
Fwiw, I think having one naming convention (dromedary case e.g. `geekdocHidden`) is the right call.  The less theme-specific naming conventions to learn about, the better

Reply:
As there were a lot more occurrences, I have created a dedicated PR for this. https://github.com/thegeeklab/hugo-geekdoc/pull/647

After it got merged, please rebase your PR.

Reply:
Rebase complete 👌